### PR TITLE
Fix minor typos and grammar issues in documentation

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

File: docs/Decorators.md [hunk 1]
```diff
@@ -2,39 +2,41 @@
...
 ```
-Decorators are not *overwritable*, if you try to declare a decorator already declared *(in other words, use the same name)*, `decorate` will throw an exception.
+
+Decorators are not *overwritable*. If you try to declare a decorator that was previously declared *(in other words, use the same name)*, `decorate` will throw an exception.
 
 <a name="decorate-reply"></a>
 **decorateReply**  
-As the name suggest, this api is needed if you want to add new methods to the `Reply` core object.  
-Just call the `decorateReply` api and pass the name of the new property and its value.
+As the name suggest, this API is needed if you want to add new methods to the `Reply` core object. Just call the `decorateReply` API and pass the name of the new property and its value:
```
Comment (jsumners) [inline docs/Decorators.md]: `suggest` => `suggests`

File: docs/Getting-Started.md [hunk 2]
```diff
@@ -55,13 +55,12 @@ fastify.listen(3000, function (err) {
   console.log(`server listening on ${fastify.server.address().port}`)
 })
 ```
-*If you want to know more about the serialization and validation, check [here](https://github.com/fastify/fastify/blob/master/docs/Validation-And-Serialize.md)!*
+
+*To learn more about using serialization and validation, check [here](https://github.com/fastify/fastify/blob/master/docs/Validation-And-Serialize.md)!*
```
Comment (jsumners) [inline docs/Getting-Started.md]: `cbeck [here]` => `see [Validation and Serialize]`

File: docs/Validation-And-Serialize.md [hunk 3]
```diff
@@ -1,63 +1,57 @@
...
 
-The `schemaCompiler` is a function that returns a function that validates the body, url parameters, headers and the query string.
-
-The default `schemaCompiler` returns a function that implements the `ajv` validation interface.
-`fastify` use it internally to speed the validation up.
+The `schemaCompiler` is a function that returns a function that validates the body, url parameters, headers, and query string. The default `schemaCompiler` returns a function that implements the `ajv` validation interface. Fastify uses it internally to speed the validation up.
 
-If you want to change the default options of the `ajv` instance you can pass the ajv option to Fastify.
-You can find all the options in the [Ajv documentation](https://github.com/epoberezkin/ajv#options).
+If you want to change the default options of the `ajv` instance, you can pass the `ajv` option to Fastify. You can find all the options in the [Ajv documentation](https://github.com/epoberezkin/ajv#options).
```
Comment (jsumners) [inline docs/Validation-And-Serialize.md]: `You can find all the options` => `The options are described in the...`

**Comment by gj** [comment]
Made those changes @jsumners !

Also, for consistency, what do you think about renaming `Validation and Serialize` to `Validation and Serialization`?

**Comment by jsumners** [comment]
@gj I am in favor of it. @fastify?

**Comment by mcollina** [comment]
Ok for me

**Comment by mcollina** [comment]
Thanks!!

**Pull request merged**